core/common/src/main/java/alluxio/util/ConfigurationUtils.java

```diff
@@ -247,4 +271,175 @@ public static boolean masterHostConfigured() {
   public static String valueAsString(String value) {
     return value == null ? "(no value set)" : value;
   }
+
+  /**
+   * Returns an instance of {@link AlluxioConfiguration} with the defaults and values from
+   * alluxio-site properties.
+   *
+   * @return The default set of Alluxio properties loaded from the site-properties file
```

Comment:
we have a concept of defaults and it doesn't include the alluxio-site.properties file. I think we need a new concept here

Reply:
I would be okay with a new term. I don't know what to call it though

Reply:
I think we could say "configuration loaded from alluxio site-properties" since configuration always incorporates default configuration